service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -101,8 +101,10 @@ public SendingAmqpLink SendingLink
 
         public async override Task CloseAsync()
         {
+            this.faultTolerantSendingLink.Dispose();
```

Comment:
Do you mean that this line should be removed because the link will be shutdown in `iotHubConnection.Dispoise()` ?